### ISSUE TITLE
crashed and closed when importing the ayat

### ISSUE THREAD

**Issue by tammarita** [rebinf/QuranVideoMaker#38]
Basically the app closes on its own after I click on the "OK" button to import the ayat
I've tried again on two computers just to make sure, with one of them having 8gb in RAM so i know that's not the issue, I even tried with shorter surah's, but to no avail. The app was working just fine for me like a week ago, so I'm puzzled. I also tried redownloading the app entirely but nothing changed.

**Comment by rebinf** [comment]
Importing? You mean when it's exporting the video?
What operating system are you using?
Is it possible to attach a sample project demonstrating the crash?

**Comment by tammarita** [comment]
No, i mean that when i want to add the ayat, i select the surah i want, and after i click on "ok", the app suddenly closes.
I'm on windows on the two computers btw.
If you don't get what I mean, it's when I'm on this tab (see the image attached below):
![Screenshot 2023-12-10 144354](https://github.com/rebinf/QuranVideoMaker/assets/147345888/5054039b-23bc-4efc-ba53-31ffe284f35f)

And I'm pretty sure it's not a RAM problem, since I tried it on the other computer with 8gb of RAM, and I made sure to restart it multiple times.
I don't know what the issue is, the app worked perfectly well for me before
